web/models/notes.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Notox.Note do
+  use Notox.Web, :model
+
+  schema "notes" do
+    field :title, :string
+    field :description, :string
+
+    timestamps
+  end
+end
```

Comment:
When I make these fixes tonight, I'm also going to add the changeset to this model. I didn't do it yesterday because it was getting late and I justified leaving it for the `:new` action. But it probably makes sense to add a basic changeset when adding a new model